apps/web/components/common/FrontendExcelUploadDialog.tsx

```
@@ -23,25 +23,59 @@ import {
-import { FileUp, Loader2, CheckCircle } from "lucide-react";
+import {
+  FileUp,
+  Loader2,
+  CheckCircle,
+  AlertTriangle,
+  Info,
+} from "lucide-react";
 import * as XLSX from "xlsx";
 import { toast } from "sonner";
 
 interface TableHeader {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TableHeader'. Consider to delete it or rename it.

'TableHeader' is defined here:


(lint/suspicious/noRedeclare)